client/src/components/faqBlock/faqconfig.json

```
@@ -1,6 +1,6 @@
 {
     "General": {
-        "Where and when is Career++ happening?": "January 18 - 19th. The remote aspect will occur on the 18th and the in-person (on the 19th) will - tentatively - be at Donadeo Innovation Center for Engineering at the University of Alberta North Campus",
+        "Where and when is Aether happening?": "January 18 - 19th. The remote aspect will occur on the 18th and the in-person (on the 19th) will - tentatively - be at Donadeo Innovation Center for Engineering at the University of Alberta North Campus",
```

Make sure the date is changed here as well
```suggestion
        "Where and when is Aether happening?": "October 10 - 11th. The remote aspect will occur on the 10th and the in-person (on the 11th) will - tentatively - be at Donadeo Innovation Center for Engineering at the University of Alberta North Campus",
```